### PR TITLE
try using preinstalled podman

### PR DESCRIPTION
**Pull request by BenTheElder**
/hold
xref: https://github.com/kubernetes-sigs/kind/issues/2070

### CODE REVIEW

**Comment by BenTheElder** [comment]
looks like we get 2.0.24 currently, though note:

> Ubuntu-latest workflows will use Ubuntu-20.04 soon. For more details, see https://github.com/actions/virtual-environments/issues/1816

**Comment by BenTheElder** [comment]
/hold cancel
the current approach fails, this works and the jobs pass using podman.

**Comment by BenTheElder** [comment]
log export flakes and prow flakes, but this works, and unblocks having CI again.

**Comment by aojea** [comment]
honestly, didn't know podman came preinstalled

File: .github/workflows/podman.yml [hunk 1]
```diff
@@ -43,13 +43,9 @@ jobs:
...
+      - name: Setup podman
         run: |
-          . /etc/os-release
-          echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/${PODMAN_VERSION}/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
-          curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/${PODMAN_VERSION}/xUbuntu_${VERSION_ID}/Release.key | sudo apt-key add -
-          sudo apt-get update -qq
-          sudo apt-get -qq -y install podman
+          podman version
           # Install network
           sudo mkdir -p /etc/cni/net.d
```
Comment (aojea) [inline .github/workflows/podman.yml:50]: do we need this too?

**Pull request merged**